fs/span-manager/span_manager_test.go

```diff
@@ -135,35 +135,43 @@ func TestSpanManagerCache(t *testing.T) {
 	cache := cache.NewMemoryCache()
 	defer cache.Close()
 	m := New(ztoc, r, cache)
-	m.addSpanToCache("spanId", content)
+	spanID := 0
+	err = m.ResolveSpan(soci.SpanId(spanID), r)
```

Comment:
any reason on why the move from `addSpanToCache` to `ResolveSpan`?

Reply:
Because ResolveSpan changes the span states which are needed for testing expected behavior of `resolveSpanFromCache`, while `addSpanToCache` just adds a span to cache and doesn't change span states.